ThermalTalk/Reliance/ReliancePrinter.cs

```diff
@@ -123,14 +127,17 @@ public override void SetFont(ThermalFonts font)
             switch (font)
             {
                 case ThermalFonts.A:
-                    internalSend(CPI11);
-                    break;
+                    Logger.Trace("Attempting to set font to font CPI11.");
+                    return internalSend(CPI11);
                 case ThermalFonts.B:
-                    internalSend(CPI15);
-                    break;
+                    Logger.Trace("Attempting to set font to font CPI15.");
+                    return internalSend(CPI15);
                 case ThermalFonts.C:
-                    internalSend(CPI20);
-                    break;
+                    Logger.Trace("Attempting to set font to font CPI20.");
+                    return internalSend(CPI20);
+                default:
+                    Logger.Trace("Invalid font selected.");
+                    return ReturnCode.ExecutionFailure;
```

Comment:
Capture all these return codes and keep a single return at the end. I'm not opposed to early returns but in this case it makes sense to keep them together.